circle.yml

```diff
@@ -6,8 +6,7 @@ dependencies:
   # Note, we used to use the naive caching of docker images, but found that it was quicker
   # just to pull each time. #rollondockercaching
   override:
-    - docker pull pelson/obvious-ci:latest_x64
-#    - docker pull pelson/conda32_obvious_ci
+    - docker pull condaforge/linux-anvil
```

Comment:
To add some meat to the above comment's bones: the condaforge docker image is a lot more sophisticated (and newer) than the obvious-ci docker image. The condaforge image also takes a number of steps to try and preserve backward-compatibility to ~RHEL6. Overall though, while newer is good, we probably don't need the extra complexity of the condaforge docker image in conda-recipes-scitools, especially if it causes some unexpected impacts.